app/assets/javascripts/discourse/tests/acceptance/create-invite-modal-test.js

```diff
@@ -242,3 +243,18 @@ acceptance(
     });
   }
 );
+
+acceptance(
+  "Invites - Populates Edit Invite Form with saved invite data",
+  function (needs) {
+    needs.user();
+
+    test("shows correct saved data in form", async function (assert) {
+      await visit("/u/eviltrout/invited/pending");
+      await click(
+        ".user-invite-list tbody tr:nth-child(1) .invite-actions .btn:nth-child(1)"
```

Comment:
```suggestion
        ".user-invite-list tbody tr:first-child .invite-actions .btn:first-child"
```

Slightly more redabable. For more readability, we could also add a class to this button in the template, something like "invite-edit" and then use it in the test.

Reply:
this could be reduced to:

```suggestion
        ".user-invite-list tbody tr .invite-actions .btn"
```